sectionsUtilityTool/src/main/webapp/assets/js/controllers/controllers.js

```diff
@@ -7,6 +7,9 @@ sectionsApp.run(function ($rootScope) {
   $rootScope.user = $.trim($('#uniqname').val());
 });
 
+function avoidCachingViaParameter(){
```

Comment:
Consider either adding the "_=" prefix to the string returned or changing the name to something like "generateCurrentTimestamp" so that the name better reflects the purpose of the method.